nservicebus/logging/index_exception-data_core_[7.2,).partial.md

```diff
@@ -0,0 +1,3 @@
+## Additional exception data
+
+Starting from NServiceBus version 7.2, exceptions from failing message handlers might contain additional error information in the `Exception.Data` property. While the default logger exposes those information automatically, other loggers might require additional configuration.
```

Comment:
Would it make sense to add a section telling users that they can add their own behaviour to add more data? Eg. adding the "tenant-id" in a multi-tenant solution?